src/cluster.c

```diff
@@ -172,24 +175,25 @@ typedef struct {
 /* Assign index to each aux field */
 typedef enum {
     af_start,
-    af_shard_id = af_start,
-    af_count,
+    af_shard_id = af_start+1,
+    af_count = 2,
 } auxFieldIndex;
 
 /* Note that
  * 1. the order of the elements below must match that of their
  *    indices as defined in auxFieldIndex
  * 2. aux name can contain characters that pass the isValidAuxChar check only */
 auxFieldHandler auxFieldHandlers[] = {
+    {"nodename", auxHumanNodenameSetter, auxHumanNodenameGetter, auxHumanNodenamePresent},
     {"shard-id", auxShardIdSetter, auxShardIdGetter, auxShardIdPresent},
```

Comment:
```suggestion
    {"shard-id", auxShardIdSetter, auxShardIdGetter, auxShardIdPresent},
    {"nodename", auxHumanNodenameSetter, auxHumanNodenameGetter, auxHumanNodenamePresent},
```

Reply:
Hello @madolson , Do we want to change the order in output as well? 
127.0.0.1:6386> cluster nodes
1811ceeec50fbe337017f5ef17cf7307bf1554b7 127.0.0.1:6386@16386,hostthreenew,**shard-id=dbb1d6860c1a7227eaf1dc9f0411f3f42d237497,nodename=nodethreenew.com**_ myself,master - 0 1681508302000 2 connected 11001-16383
3ba3de2ae177c060c99c90d753049b25fef3ae93 127.0.0.1:6384@16384,hostonenew,shard-id=2752cbe896d6ab7cd35a7f1229b46347b05fb078,nodename=nodeonenew master - 0 1681508304281 1 connected 0-5500
812dfd3fe60b268830dfc11e284d08bd03b1142e 127.0.0.1:6385@16385,,shard-id=171b4d57024bf7e9660e047437ed4c27498d1562,nodename=nodetwonew master - 0 1681508303279 0 connected 5501-11000